### PR TITLE
build-prep-1node.sh: improve to use by non-root users

### PR DESCRIPTION
**Pull request by andriytk**
It's still required that the user have sudo privileges to run the script though.

### CODE REVIEW

**Comment by nezzzu** [comment]
Also, if we can have 3 pools and if the hctl status output resembles that of Sage cluster then same motraddr.sh script in m0client-sample-apps can be used. @andriytk ?

**Comment by nezzzu** [comment]
A further improvement would be to completely skip the build step if the required rpms are already installed on the system (or just an option to skip the build step) and go directly to single node setup steps. This will probably require the singlenode.yaml contents incorporated in the script if they are not rpm installed by default. @andriytk ?

**Comment by andriytk** [comment]
@nezzzu this is devvm - for development purposes, not just install and run.

m0client-sample-apps can be used even now without multiple pools, one pool is enough. The rc-files for the utilities can be created manually.

Not familiar with motraddr.sh script, cannot advise on that.

**Review by huanghua78 — COMMENTED** [review]
13  ./scripts/install-motr-service

21 make && make devinstall

also need `sudo`, I think.

**Comment by andriytk** [comment]
Yes, thanks @huanghua78 .

**Review by huanghua78 — APPROVED** [review]

**Comment by andriytk** [comment]
@nezzzu as you suggested - it builds and installs rpms by default now. The development mode is still available via the `-dev` cmdline option.

Give it a try.

File: scripts/provisioning/build-prep-1node.sh [hunk 1]
```diff
@@ -1,31 +1,72 @@
...
 # Clone, build and prepare to start a single-node cluster.
+#
+# Usage: ./build-prep-1node.sh [-dev]
+#
+#   -dev  development mode, don't build and install rpms
+#
+set -e -o pipefail
 
-set -eu -o pipefail
+dev_mode=
```
Comment (huanghua78) [inline scripts/provisioning/build-prep-1node.sh:11]: Add comments like:
# set this to "YES" to build source code in local dir and then install only motr services
# and leave it empty to build RPMs and install them.

**Review by huanghua78 — APPROVED** [review]

File: scripts/provisioning/build-prep-1node.sh (hunk 1 above)
Comment (andriytk) [reply scripts/provisioning/build-prep-1node.sh:11]: Hmm, isn't it enough what's written in the "Usage" above - "development mode, don't build and install rpms"?

**Review by madhavemuri — APPROVED** [review]

**Pull request merged**